Library/Formula/kibana.rb

```diff
@@ -0,0 +1,71 @@
+class Kibana < Formula
+  homepage 'https://www.elastic.co/products/kibana'
+  url 'https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz'
+  version '4.0.2'
+  sha1 'ce6136d826cf8ac1da71abea419f2f5220055933'
+  head 'https://github.com/elastic/kibana.git'
+
+  def install
+    # install base files
+    prefix.install %w[ node src ]
+
+    # setup etc path
+    etc_path.mkpath
+
+    # setup var path
+    var_path.mkpath
+
+    # copy/link config
+    cp 'config/kibana.yml', conf_path unless conf_path.exist?
```

Comment:
Pretty sure you can get away with doing `(etc/"kibana").install "kibana.yml"` and that way you can drop the `etc_path`, `conf_path` logic.